components/org.wso2.carbon.identity.conditional.auth.functions.user/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/user/SetAccountAssociationToLocalUser.java

```diff
@@ -0,0 +1,34 @@
+package org.wso2.carbon.identity.conditional.auth.functions.user;
+/*
+ *  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Copyright goes to as the first line.

Reply:
fixed in 97e9bbd7c06d24eca3cfde4e5b5428a54ede4f30

Reply:
fixed in 97e9bbd7c06d24eca3cfde4e5b5428a54ede4f30